templates/Alloy.Mvc/Views/Shared/Blocks/TeaserBlockWide.cshtml

```diff
@@ -11,8 +11,8 @@
             Url.PageLinkUrl(Model.Link),
             Model.Heading))
         {
```

Comment:
We could replace this HTML helper with a tag helper.

Reply:
This is a custom HtmlHelper method in Alloy, and I think it is out of scope for this story.

Reply:
Maybe :) But I thought the story was to replace HTML helpers.